test/gtest/uct/test_uct_ep.cc

```diff
@@ -240,4 +277,5 @@ UCS_TEST_SKIP_COND_P(test_uct_ep, destroy_entity_after_send,
     }
 }
 
-UCT_INSTANTIATE_NO_SELF_TEST_CASE(test_uct_ep)
+UCT_INSTANTIATE_TEST_CASE(test_uct_ep)
+UCT_INSTANTIATE_CUDA_IPC_TEST_CASE(test_uct_ep)
```

Comment:
maybe use UCT_INSTANTIATE_CUDA_TEST_CASE?

Reply:
it's mem_type only at the moment:
https://github.com/openucx/ucx/blob/4655c0445fe5fcce9e4eeffbaea1b8f35baf9d5d/test/gtest/uct/test_uct_perf.cc#L282

can instead rename CUDA -> CUDA_MEMTYPE and then add cuda_ipc to CUDA_TEST_CASE

Reply:
or even better -> rename UCT_TEST_TLS to UCT_TEST_NO_CUDA_IPC_TLS
and then add new UCT_TEST_TLS with cuda_ipc included.

Reply:
Why not instantiate on cuda_ipc as well?

Reply:
It causes failure in some unrelated tests

Reply:
I guess we can use UCT_TEST_NO_CUDA_IPC_TLS, but it's not expected cuda_ipc would fail, need to understand why